client/src/components/Donation/DonationModal.js

```diff
@@ -98,6 +104,16 @@ function DonateModal({
     }
   };
 
+  const handleModalHide = () => {
+    if (location) {
+      // Might not be necessary, but still nothing wants to listen to me...
+      const to = location.pathname + location.hash;
+      // TODO: Does not work...probably not the best approach.
+      console.log(to, location);
+      navigate('#claim-cert-block');
+    }
```

Comment:
```suggestion
    goToAnchor('claim-cert-block');
```
It took a while to get to the bottom of this, but this seems to do the trick.

My guess is that `navigate` tries to go to `#claim-cert-block`, but that id doesn't exist (`react-scrollable-anchor` never creates an id and just does some JS magic, I guess)